Sources/BraintreePayPal/BTPayPalClient.swift

```diff
@@ -62,9 +62,6 @@ import BraintreeDataCollector
     /// Used for analytics purposes, to determine if brower-presentation event is associated with a locally cached, or remotely fetched `BTConfiguration`
     private var isConfigFromCache: Bool?
 
-    /// Used for sending the type of flow, universal vs deeplink to FPTI
-    private var linkType: LinkType?
-
```

Comment:
We will still want to keep the link type for the PayPal flow, but we can remove it from Venmo.

Reply:
Oh ok, I'll need to add the LinkType enum back in then

Reply:
LinkType is a little confusing to me. Does it represent the type of the URL that we are opening? Or the URL that returns back to the SDK? Might be worth clarifying in the docstring

Reply:
It's the type of URL link we're opening when a buyer clicks to purchase with __ (PayPal in this case). I'll clarify in the docstring